### PR TITLE
Fixed an issue with coffee beans from Cargo.

### PR DESCRIPTION
**Pull request by MarinaGryphon**
Why it wasn't caught in review, I don't know. 😛

### CODE REVIEW

**Comment by skull132** [comment]
While you're here. @MarinaGryphon remove the snowflake characters from that one coffee name. Ref #3152 .

**Comment by MarinaGryphon** [comment]
I think I can just change the encoding and make it work?

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 1]
```diff
@@ -2450,10 +2450,10 @@
 	glass_desc = "A mug of a rich strong roast, you think it could be a lot better if someone added something extra to it."
 
 /datum/reagent/drink/white_coffee
-	name = "Café Au Lait"
+	name = "Cafe Au Lait"
 	id = "white_coffee"
 	description = "A fancy name for something thats just coffee and milk."
-	color = "#482000"
+	color = "##A64D07"
```
Comment (skull132) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: Double hash-tag.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm [hunk 2]
```diff
@@ -2463,18 +2463,18 @@
...
 	..()
 	M.heal_organ_damage(0.5 * removed, 0)
 
 /datum/reagent/drink/cafe_melange
-	name = "Café Mélange"
+	name = "Cafe Melange"
 	id = "cafe_melange"
 	description = "A delicious mug of creamy coffee."
-	color = "#482000"
+	color = "##A64D07"
```
Comment (skull132) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm]: double hashtag.

File: code/modules/reagents/dispenser/coffeebeans.dm [hunk 3]
```diff
@@ -2,6 +2,7 @@
 	name = "A jar of Coffee Beans "
 	desc = "This goes into a coffee maker!"
 	label = "Spesscafe Dark Blend"
+	icon = 'icon/obj/drinks.dmi'
```
Comment (skull132) [inline code/modules/reagents/dispenser/coffeebeans.dm]: `icons/obj/drinks.dmi` no?

**Pull request merged**